docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -29,12 +28,12 @@
     "\n",
     "## Prerequisites\n",
     "\n",
-    "### Enable model monitoring on the project\n",
+    "### Set the datastore profiles and the model monitoring credentials\n",
     "\n",
-    "Enable model monitoring for a project with [`enable_model_monitoring`](https://docs.mlrun.org/en/stable/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.enable_model_monitoring).\n",
-    "The controller runs, by default, every 10 minutes, which is also the minimum interval. \n",
-    "You can modify the frequency with the parameter `base_period`. \n",
-    "To change the `base_period`, call `update_model_monitoring_controller`. "
+    "Before you enable model monitoring, you must configure two datastore profiles: TSDB and streaming. A datastore profile is a named set of configurations (potentially including secrets) that is used in running pods. \n",
+    "Model monitoring supports [Kafka](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.DatastoreProfileKafkaSource) and [V3IO](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.DatastoreProfileV3io) as streaming platforms, and [TDEngine](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.TDEngineDatastoreProfile) and [V3IO](https://docs.mlrun.org/en/stable/api/mlrun.datastore/index.html#mlrun.datastore.datastore_profile.DatastoreProfileV3io) as TSDB platforms. \n",
```

Comment:
why not using `{py:class}` directive? [example](https://github.com/mlrun/mlrun/blob/0860252de15abb05e84d4b34ffb78d7222cc588c/docs/store/artifacts.md#L71)

Reply:
Because the tutorials have to be standalone docs - not only as part of the doc build